Assets/MixedRealityToolkit.Services/InputSystem/GazeProvider.cs

```diff
@@ -246,7 +255,7 @@ public override bool TryGetPointerRotation(out Quaternion rotation)
             /// <param name="inputSource"></param>
             public void RaisePointerDown(MixedRealityInputAction mixedRealityInputAction, Handedness handedness = Handedness.None, IMixedRealityInputSource inputSource = null)
             {
-                MixedRealityToolkit.InputSystem.RaisePointerDown(this, mixedRealityInputAction, handedness, inputSource);
+                gazeProvider.InputSystem?.RaisePointerDown(this, mixedRealityInputAction, handedness, inputSource);
```

Comment:
Can we just pass in InputSystem to the InternalGazePointer?

Reply:
we may be able to. that would be outside of the scope of this PR. the goal is to limit the change as much as possible.